docs/chart_tests.md

```diff
@@ -73,11 +71,13 @@ STATUS: DEPLOYED
 ```
 
 2. `$ helm test quirky-walrus`
+
 ```
 RUNNING: quirky-walrus-credentials-test
 SUCCESS: quirky-walrus-credentials-test
 ```
 
 ## Notes
+
```

Comment:
Maybe we could add in a line about how we still support the test-success annotation temporarily as we transition over to the test annotation under notes for clarity?

Reply:
Docs no longer live here... will have to do a follow up for helm/helm-www later. 😄